modules/nextflow/src/main/groovy/nextflow/executor/FluxExecutor.groovy

```diff
@@ -61,7 +61,12 @@ class FluxExecutor extends AbstractGridExecutor {
         List<String> result = ['flux', 'mini', 'submit']
         result << '--setattr=cwd=' + quote(task.workDir)
         result << '--job-name="' + getJobNameFor(task) + '"'
-        result << '--output=' + quote(task.workDir.resolve(TaskRun.CMD_LOG))  // -o OUTFILE
+
+        // Only write output to file if user doesn't want written entirely to terminal
+        Boolean terminalOutput = task.config.navigate('terminalOutput') as Boolean
```

Comment:
```suggestion
        Boolean terminalOutput = task.config.navigate('flux.terminalOutput') as Boolean
```

Reply:
I'm wondering - how did this work before with  just terminalOutput when it was under process.terminalOutput? Does navigate do a search of sub-fields? :thinking: